sui_types/src/unit_tests/utils.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
shouldn't this file be part of the crate `test-utils`? Also it is probable that some of these utils are already implemented there

Reply:
Ok, I actually do not really know how to do this. I tried doing it, but then the same type appeared as different types :(